### DISCUSSION TITLE
Improvement For the Document

### DISCUSSION THREAD

**Discussion by xianshenglu** [selfrefactor/rambda#758] [Ideas]
Hi,

I'm trying to find lodash alternative and meet this library. This is the document page I see.

![image](https://github.com/user-attachments/assets/62acbe58-4441-452c-98fe-ee2a5b07efa4)

From my point of view maybe the document is kind of too simple or incomplete?

For example, there's no function signature, description and example for `addIndex`.

It does have example link. However, it's not convenient in my opinion. I need to click the link and wait for 2 seconds and then navigate back again. 

Compare with other similar libraries

![image](https://github.com/user-attachments/assets/f88d77ba-d592-4066-92c0-d16aeecac962)
![image](https://github.com/user-attachments/assets/5d36b2ec-fb7e-4abc-a090-351fd61c54bd)
![image](https://github.com/user-attachments/assets/c5c84e1c-7529-4ee3-b56c-66ff9a678fb2)

Anyway still thanks for the great work for building this library.

**Comment by selfrefactor** [comment]
I see your point and I will try to make adjustments.
Main issue is that there is limit of how big README.md can be and that is what I use to build `docsify` site. I will try to have separate file for that and I will keep you updated.

Thank you for the appreciation.

**Comment by selfrefactor** [comment]
Hi

one part of solving this is to have script to build docs site. Now it looks like this

![image](https://github.com/user-attachments/assets/f527c6a9-8e4d-4f6e-9459-506b87dfc456)

What is left is to add description to each method, but this is ongoing task that I cannot say when it will be ready, so I will be closing this ticket and thanks for bringing the issue to my attention.

**Reply by xianshenglu** [reply]
Thanks for the update!
